config/config.go

```diff
@@ -36,6 +42,12 @@ func Get() (*Config, error) {
 		GracefulShutdownTimeout:    5 * time.Second,
 		HealthCheckInterval:        30 * time.Second,
 		HealthCheckCriticalTimeout: 90 * time.Second,
+		EnablePrivateEndpoints:     true,
+		ZebedeeURL:                 "http://localhost:8082",
+		ServiceAuthToken:           "",
+		EnablePermissionsAuth:      false,
+		// NOTE: ensure that this is routed to the *unauthenticated* dataset endpoint.
+		DatasetAPIURL: "http://localhost:22000",
```

Comment:
There isn't a separate endpoint that is configured here, it just depends on how the service is set up (either on web or publishing) so this comment can be removed

Reply:
will remove, but given the slightly roundabout way in which we are trying to get published datasets, I would provide as much info as possible, maybe rephrase the above more in line with your comment